src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -457,6 +462,8 @@
     <ReadEmccProps JsonFilePath="$(_WasmRuntimePackSrcDir)emcc-props.json">
       <Output TaskParameter="EmccProperties" ItemName="_EmccPropItems" />
       <Output TaskParameter="WasmOptConfigurationFlags" ItemName= "_WasmOptConfigurationFlagsItems" />
+      <Output TaskParameter="EmccExportedFunctions" ItemName= "EmccExportedFunction" />
+      <Output TaskParameter="EmccExportedRuntimeMethods" ItemName= "EmccExportedRuntimeMethod" />
```

Comment:
And the item names here too - `_DefaultEmccExportedFunction`, and `_DefaultEmccExportedRuntimeMethod`.